docker.d/pre-stop.sh

```diff
@@ -10,7 +10,14 @@ SCRIPTWORKER_PID=${SCRIPTWORKER_PID:-1}
 # in case the task takes too long, this script exits 2 minutes before
 # `terminationGracePeriodSeconds` to let scriptworker upload files and report
 # `machine-shutdown` to Taskcluster.
-POLL_DURATION=1080
+case ${PROJECT_NAME} in
+    tree)
+        POLL_DURATION=3480
```

Comment:
Nit: let's add a comment to remind ourselves about the value duplication

```suggestion
        # Keep this value in sync with [1] so that it's 2 minutes shorter than the terminationGracePeriodSeconds
        # [1] https://github.com/mozilla-services/cloudops-infra/blob/d94d5fd6a7704ffd2c829d870206f5c0ed8d75e7/projects/relengworker/k8s/charts/tree/templates/deployment.yaml#L35
        POLL_DURATION=3480
```

Reply:
That feels like a bit of a repeat of the existing comment?

Reply:
A little bit. Although, the comment above doesn't say where `terminationGracePeriodSeconds` lives and is not super clear that we must manually keep them in sync. We could update the comment above if that helps having a single bloc of text.